ui/src/pages/ZoomableBoard.js

```diff
@@ -105,6 +105,24 @@ export default function ZoomableBoard({ replayMode }) {
     }),
     []
   );
+  const handleTileClick = useCallback(
+    memoize((coordinate) => {
+      console.log("Clicked Tile ", coordinate);
+      if (state.isMovingRobber) {
+        const matchingAction = state.gameState.current_playable_actions.find(action =>
+          action[1] === "MOVE_ROBBER" &&
+          action[2][0].every((val, index) => val === coordinate[index])
```

Comment:
NIT: Maybe saying `find([_, action_type, action_value] =>` might help in readability? Or adding a small comment on this second condition? I know I have code like this in a bunch of places 😅 , but it took me a while to understand what we were after here.. 👍

Reply:
Agreed, I'll destructure the actions into descriptive variable names instead